RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -697,3 +697,142 @@ boolean LX_meatMaid()
 
 	return false;
 }
+
+string LX_getDesiredWorkshed(){
+	string currentWorkshed = get_property("auto_workshed").to_lower_case();
+	//return the actual item name in case a shorthand is used
+	switch(currentWorkshed)
+	{
+		case "train":
+			return "model train set";
+		case "model train set":

```

Comment:
Have you tried setting it to "model train set"? I'm pretty sure this should moved to directly under `case "train"`. I expect current behavior to use cold medicine cabinet if you use the setting "model train set". Similar comment for other cases below

Reply:
Definitely correct on this. Assumed case worked differently than it did but after looking at the Hewn Spoon code and the Switch on the KoLMafia Wiki, I definitely see that I had it wrong initially.